pymobiledevice3/cli/developer.py

```diff
@@ -271,6 +277,32 @@ def screenshot(service_provider: LockdownClient, out):
         out.write(Screenshot(dvt).get_screenshot())
 
 
+def app_lookup(install_service: InstallationProxyService, bundle_id: str):
+    # install_service.lookup is not working for RSD，use browse instead
+    # will get "ConnectionAbortedError" if use lookup
+    app_infos = [v for v in install_service.browse() if v["CFBundleIdentifier"] == bundle_id]
+    if len(app_infos) == 0:
+        raise RuntimeError(f'No app with bundle id {bundle_id} found')
+    return app_infos[0]
+
+
+@dvt.command('xcuitest', cls=Command)
+@click.option("--bundle-id", required=True, help="Bundle ID of the app to test")
+def xcuitest(service_provider: LockdownClient, bundle_id: str):
+    """ start XCUITest """
+    # python -m pymobiledevice3 developer dvt xcuitest --bundle-id com.facebook.WebDriverAgentRunner.xctrunner
+    # python -m pymobiledevice3 developer dvt xcuitest --bundle-id com.facebook.WebDriverAgentRunner.xctrunner --tunnel $UDID
+    try:        
+        install_service = InstallationProxyService(lockdown=service_provider)
+        app_info = app_lookup(install_service, bundle_id)
+
+        afc = HouseArrestService(lockdown=service_provider, bundle_id=bundle_id, documents_only=False)
+        
+        XCUITestService(service_provider, afc, app_info).run(bundle_id)
+    except:
```

Comment:
It's a bad habit to just ignore everything. If people get errors they won't be able to report them. please remove this